index.html

```diff
@@ -77,7 +77,165 @@ <h3 class="username">Karma</h3>
         </div>
     </div>
     <div class="content">
-      <h3>content</h3>
+      <div class="content-header">
+        <h2 class="content-header-item">Home</h2>
+        <i class="fa fa-certificate" title="Top Tweets"></i>
+      </div>
+      <div class="what-happening-box">
+        <div class="tweet-box">
+          <a href="#" class="user-card">
+            <img src="images/avatar.png" alt="avatar" class="user-avatar"/>
+          </a>
+          <div class="text-privacy-container">
+            <textarea name="tweet" class="tweet-field" cols="40" rows="1" placeholder="What's happening?" required></textarea>
+            <div class="hint">
+              <a href="#"  class="hint-message">
+                  <i class="fa fa-globe"></i> Everyone can reply
+              </a>
+            </div>
+          </div>
+        </div>
+        <div class="options">
+          <div class="icons">
+            <a href="#" class="icon" title="Media"><i class="fa fa-image" ></i></a>
+            <a href="#" class="icon" title="GIF"><i class="fa fa-upload" ></i></a>
+            <a href="#" class="icon" title="Poll"><i class="fa fa-bars" ></i></a>
+            <a href="#" class="icon" title="Emoji"><i class="fa fa-comment" ></i></a>
+            <a href="#" class="icon" title="Schedule"><i class="fa fa-file" ></i></a>  
```

Comment:
Tip: you can make the `i` icon in a different way as it doesn't have content inside it 

example: `<i class="fa fa-image" />`

Reply:
by searching i found self closing to tag "i" not prefered  or allowed in HTML up to and including HTML 4.01, <i class="icon-calendar btn"/> is formally wrong syntax

source: https://stackoverflow.com/questions/16253890/self-closing-an-i-html-tag-or-not